NEWS.md

```diff
@@ -238,6 +238,9 @@ Library improvements
 
   * The functions `strip`, `lstrip` and `rstrip` now return `SubString` ([#22496]).
 
+  * The constructor of `SubString` now checks if the requsted view range
```

Comment:
Shouldn't this be listed under "breaking changes", since it can raise an error for previously working cases?

Reply:
moved, although if someone used an index outside a valid range it was probably a bug

Reply:
Sure, but that can still break code since no error was thrown before. Anyway, looks good to me now.